web/app/components/dashboard/latest-updates.hbs

```diff
@@ -42,7 +42,6 @@
           @owner={{get doc.owners 0}}
           @productArea={{doc.product}}
           @status={{lowercase doc.status}}
-          @thumbnail={{doc.googleMetadata.thumbnailLink}}
```

Comment:
Hopefully we're going to add thumbnails back sooner than later, but I'll defer to you if you want to remove this before then.